Tools-Override/FrameworkTargeting.targets

```diff
@@ -4,10 +4,10 @@
   <ItemGroup>
     <TargetingPackDirs Include="$(RefPath)" />
     <AdditionalReferencePaths Include="@(TargetingPackDirs)" />
+    <Reference Condition="($(NuGetTargetMoniker.StartsWith('.NETStandard')) OR '$(IsReferenceAssembly)' != 'true') AND Exists('$(RefPath)netstandard.dll') AND '$(MSBuildProjectExtension)' != '.pkgproj'" Include="netstandard" />
```

Comment:
Yeah, I don't like this either, wasn't sure if there might even be a better place for this to live. (eg: targetingpacks.props.)  For now I'll just polish it up a little.